style.css

```diff
@@ -236,3 +229,94 @@ nav {
   grid-template-columns: 1fr;
   background-color: #e5e5e5;
 }
+
+.contact-form {
+  background-image: url(./images/background2.png);
+  background-color: #6070ff;
+  background-size: cover;
+}
+
+.contact-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-info h2 {
+  font-size: 40px;
+  text-align: center;
+  color: white;
+  margin-top: 160px;
+  margin-bottom: 20px;
+}
+
+.contact-info p {
+  font-size: 20px;
+  text-align: center;
+  margin-bottom: 38px;
+  padding: 0 70px;
+  color: white;
+}
+
+.contact-input {
+  width: 317px;
+  height: 38px;
+  margin-bottom: 20px;
+  margin-left: 24px;
+  margin-right: 24px;
+  border-radius: 8px;
+  font-size: 17px;
+  border: none;
+  color: #172b4d;
+  padding: 10px 15px;
+}
+
+.contact-textarea {
+  width: 317px;
+  height: 160px;
+  margin-bottom: 20px;
+  margin-left: 24px;
+  margin-right: 24px;
+  border-radius: 8px;
+  font-size: 17px;
+  border: none;
+  color: #172b4d;
```

Comment:
- [ OPTIONAL]

Nice web page you have got 💯. However, the color of the text area is `#b3bac5;` and not `#172b4d`. Consider adding the correct color so that your design matches that of your chosen template 😄

Reply:
Thanks